iconrpcserver/server/rest_server.py

```diff
@@ -128,19 +127,28 @@ def serve(self, api_port):
         self.__app.run(host='0.0.0.0', port=api_port, debug=False, ssl=self.ssl_context)
 
 
+def get_status_by_earlgrey(channel_name) -> dict:
+    channel_stub = StubCollection().channel_stubs[channel_name]
+    status_data: dict = channel_stub.sync_task().get_status()
```

Comment:
async task is better

Reply:
How about renaming 'get_status_by_earlgrey' to 'get_channel_status' or something else?

Reply:
@zsaladin 
> async task is better
Thank you! I applied async and await keywords.

@windies21 
> How about renaming 'get_status_by_earlgrey' to 'get_channel_status' or something else?
Reasonable name. Thank you!